phpstan.neon.dist

```diff
@@ -9,13 +9,17 @@ parameters:
         - Neos.ContentGraph.PostgreSQLAdapter/src
         - Neos.ContentRepository.BehavioralTests/Classes
         - Neos.ContentRepository.Core/Classes
+        - Neos.ContentRepository.DbalTools
         - Neos.ContentRepository.Export/src
         - Neos.ContentRepository.LegacyNodeMigration/Classes
         - Neos.ContentRepository.NodeAccess/Classes
         - Neos.ContentRepository.NodeMigration/src
         - Neos.ContentRepository.StructureAdjustment/src
         - Neos.ContentRepository.TestSuite/Classes
         - Neos.ContentRepositoryRegistry/Classes
+        - Neos.ContentRepositoryRegistry.DoctrineDbalClient
+        - Neos.ContentRepositoryRegistry.PostgresDbalClient
```

Comment:
```suggestion
```

they have been removed again, my change can thus be reverted

Reply:
I'll take care of the rebasing once my three cleanups #5073 #5074 #5075 are in